src/_bentoml_impl/worker/service.py

```diff
@@ -186,6 +186,7 @@ def main(
     BentoMLContainer.development_mode.set(development_mode)
 
     server_context.service_name = service.name
+    server_context.service_apis = [api for api in service.apis.keys()]
```

Comment:
```suggestion
    server_context.service_apis = [api.route for api in service.apis.values()]
```
Make it consistent since the request_in_total metric is collected with the url path(route) not the url name

Reply:
Ah yes. Fixed, and changed the name to `service_routes`.